src/kirby/components/chip/chip.component.tns.scss

```
@@ -0,0 +1,12 @@
+@import '../../scss/native-imports';
+$min-chip-width: 50;
+.chip {
+  border-radius: 25;
+  margin-right: 10;
+  padding:10 15;
+  font-size: font-size('xs');
+  min-width: $min-chip-width;
+  &.selected {
```

Keep naming on sync:
```suggestion
  &.is-selected {
```

---

Done